scripts/ci/add_supported_release.py

```diff
@@ -72,18 +69,23 @@ def add_release_version(image: str, version: str) -> None:
 def main() -> int:
     parser = argparse.ArgumentParser()
     parser.add_argument(
-        "--image", help="image to add a new supported version", type=str
+        "--image-name", help="image to add a new supported version", type=str
     )
     args = parser.parse_args()
 
-    if args.image not in VALID_IMAGES:
-        raise ValueError("Image {} not supported".format(args.image))
+    if args.image_name not in VALID_IMAGES:
+        print(
+            "Image {} not supported. Not adding release version.".format(
+                args.image_name
+            )
+        )
+        return 0
```

Comment:
do we want to exit here with code 0 or an error code?

Reply:
right good catch, the reason for a 0 here rather than error, is that we run this task for every image we are going to release. This will update the release as a support version (for agent only currently), but we need this to be a no-op rather than failure for the other images.

This task will currently only fail if there is an error when interacting with the release database in Atlas.